docs/guides/usage.md

```diff
@@ -2,195 +2,284 @@
 
 ## Starting the server
 
-Compass interfaces with an elasticsearch cluster. Run compass using:
+Compass interfaces with a postgres and elasticsearch cluster. Run compass using:
```

Comment:
similar with `querying.md`, should we offload this information to `installation.md`?

Reply:
yes we could.